docs/data-guidelines.md

```diff
@@ -46,9 +46,9 @@ For example, the `ImageData` constructor, `ImageData()`, is represented as `api.
 
 ## DOM events (`eventname_event`)
 
-Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
+Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`. Each event's record is a member of the interface which is its target. If an event can be sent to multiple interfaces, you need to replicate the event in each of those interfaces.
```

Comment:
Can we present this an imperative? It's not optional, after all. Like this maybe?

```suggestion
Add DOM events as features of their target interfaces. If an event can be sent to multiple interfaces, then add the event as a feature of each interface. Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
```

Reply:
I've rewritten the paragraph; hopefully it satisfies. It's not the same as what you wrote but is somewhat closer. :)